src/features/MoviesCards/components/MoviesCard/MoviesCard.jsx

```diff
@@ -1,21 +1,43 @@
+/* eslint-disable jsx-a11y/interactive-supports-focus */
+/* eslint-disable no-restricted-globals */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
 /* eslint-disable jsx-a11y/img-redundant-alt */
 import clsx from 'clsx'
-import React from 'react'
+import React, { useContext } from 'react'
+import { Context } from '../../../../contexts/Context'
 import styled from './styles.module.scss'
 
 // eslint-disable-next-line object-curly-newline
-function index({ src, name, type, year }) {
+function index({ item, src, name, type, year }) {
+  const { setMovie } = useContext(Context)
+
   return (
-    <div>
-      <img className={styled.image} src={src} alt="image" />
+    <div className="col-lg-4 col-lg-3 col-md-6 col-sm-12 col-xs-12 px-5">
+      <img
+        onClick={() => {
+          scroll(0, 0)
+          setMovie(item)
+        }}
+        className={clsx(styled.image, styled.cursor)}
+        src={src}
+        alt="image"
+      />
       <br />
       <br />
```

Comment:
please don't use br tag, it's depricated

Reply:
ohh yeah u said that before too, I'll makkke this thing solved soon